core/src/main/scala/org/locationtech/rasterframes/ref/RasterRef.scala

```diff
@@ -34,35 +34,49 @@ import org.locationtech.rasterframes.tiles.ProjectedRasterTile
  *
  * @since 8/21/18
  */
-case class RasterRef(source: RFRasterSource, bandIndex: Int, subextent: Option[Extent], subgrid: Option[Subgrid]) extends ProjectedRasterTile {
+case class RasterRef(source: RFRasterSource, bandIndex: Int, subextent: Option[Extent], subgrid: Option[Subgrid], bufferSize: Short) extends ProjectedRasterTile {
```

Comment:
Why do you need both a `subgrid` and a `bufferSize`?

Reply:
@metasim I think subgrid is to read the window (that is the old API), the buffer is the size of the actual underlying this window source.